gulpfile.js

```diff
@@ -82,6 +82,11 @@ function buildTask() {
 
   var errorHandler = function (err) {
     util.log(util.colors.red('[Error]'), err.toString());
+    var browserError = 'console.error("Gulp: ' + err.toString() + '")';
+    fs.writeFileSync(outDir+'Chart.js', browserError);
+    fs.writeFileSync(outDir+'Chart.min.js', browserError);
+    fs.writeFileSync(outDir+'Chart.bundle.js', browserError);
+    fs.writeFileSync(outDir+'Chart.bundle.min.js', browserError);
```

Comment:
@simonbrunel if possible, i would like to be able to see the errors directly in the browser. I prefer to work in the editor and the browser only, and not have to check the terminal.